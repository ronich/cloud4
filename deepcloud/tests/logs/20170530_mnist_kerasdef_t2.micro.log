Downloading data from https://s3.amazonaws.com/img-datasets/mnist.npz
    8192/11490434 [..............................] - ETA: 0s 1900544/11490434 [===>..........................] - ETA: 0s 4358144/11490434 [==========>...................] - ETA: 0s 6823936/11490434 [================>.............] - ETA: 0s 8912896/11490434 [======================>.......] - ETA: 0s10887168/11490434 [===========================>..] - ETA: 0sTrain on 60000 samples, validate on 10000 samples
Epoch 1/12
367s - loss: 2.2319 - acc: 0.3242 - val_loss: 0.9810 - val_acc: 0.6962
Epoch 2/12
350s - loss: 1.4571 - acc: 0.5237 - val_loss: 1.0598 - val_acc: 0.7468
Epoch 3/12
349s - loss: 1.3167 - acc: 0.5980 - val_loss: 0.8872 - val_acc: 0.8303
Epoch 4/12
361s - loss: 1.2809 - acc: 0.6213 - val_loss: 0.8223 - val_acc: 0.8029
Epoch 5/12
383s - loss: 1.2561 - acc: 0.6330 - val_loss: 0.9567 - val_acc: 0.8164
Epoch 6/12
1541s - loss: 1.2528 - acc: 0.6390 - val_loss: 0.8804 - val_acc: 0.8333
Epoch 7/12
3953s - loss: 1.2379 - acc: 0.6432 - val_loss: 0.8289 - val_acc: 0.8462
Epoch 8/12
3870s - loss: 1.2790 - acc: 0.6488 - val_loss: 0.8009 - val_acc: 0.8094
Epoch 9/12
