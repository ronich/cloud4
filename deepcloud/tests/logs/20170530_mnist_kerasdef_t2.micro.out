epoch,acc,loss,val_acc,val_loss
0,0.324216666651,2.23185570151,0.6962,0.980966785049
1,0.523666666635,1.4570505998,0.7468,1.05981933727
2,0.598049999968,1.31670457948,0.8303,0.88716698761
3,0.621266666667,1.28091920389,0.8029,0.8222631464
4,0.633033333302,1.25605236524,0.8164,0.956662384796
5,0.639016666698,1.25280549399,0.8333,0.880396054077
6,0.643166666667,1.23788550129,0.8462,0.828938193417
7,0.648849999968,1.27895313676,0.8094,0.800878892136
